src/enhanced-button.jsx

```diff
@@ -285,6 +285,13 @@ const EnhancedButton = React.createClass({
       cursor: disabled ? 'default' : 'pointer',
       textDecoration: 'none',
       outline: 'none',
+      /*
+        This is need so that ripples do not bleed
+        past border radius.
+        See: http://stackoverflow.com/questions/17298739/
```

Comment:
See [here](http://stackoverflow.com/questions/17298739/css-overflow-hidden-not-working-in-chrome-when-parent-has-border-radius-and-chil)